test/File/RenameUploadTest.php

```diff
@@ -133,11 +133,15 @@ public function testOptions()
             'use_upload_name' => true,
             'overwrite'       => true,
             'randomize'       => true,
+            'stream_factory'  => $this->prophesize(StreamFactoryInterface::class)->reveal(),
```

Comment:
The topic of this pull request is "PSR-7 file upload parameter naming consistency." – please use a separate pull request for changing the tests. (And if you add changes that are not a part / topic of the current commit, Git's `cherry-pick` command can not be used.)

Thanks!